app/src/main/java/com/strv/movies/MainActivity.kt

```diff
@@ -46,13 +44,94 @@ class MainActivity : ComponentActivity() {
                                 DarkLightModeSwitchIcon(isDarkTheme = isDarkTheme)
                             }
                         )
+                        Greeting("MOVIES")
+                        LogInMoviesApp()
                         MoviesList(movies = OfflineMoviesProvider.getMovies())
                     }
                 }
             }
         }
     }
 
+    @OptIn(ExperimentalFoundationApi::class)
+    @Composable
+    fun LogInMoviesApp() {
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier
+                .padding(start = 40.dp, bottom = 10.dp, top = 18.dp, end = 20.dp)
+                .width(300.dp)
+        ) {
+            LogInElements()
+            otherElementsForLogin()
+        }
+    }
+
+    @Composable
+    private fun otherElementsForLogin() {
+        Button(
+            onClick = {},
+            modifier = Modifier
+                .width(280.dp)
+                .padding(bottom = 10.dp)
+            ) {
+            Icon(
```

Comment:
You can wrap the `Icon` and `Text` here in `Row` to be sure they will be correctly placed next to each other - in a row (not overlapping, etc.).